test/unit/lib/static/components/section/title/browser.jsx

```diff
@@ -103,7 +103,7 @@ describe('<BrowserTitle/>', () => {
             const browsersStateById = {'yabro': {checkStatus: UNCHECKED}};
             const tree = mkStateTree({browsersById, resultsById, browsersStateById});
 
-            const component = mkBrowserTitleComponent({browserId: 'yabro', browserName: 'yabro'}, {tree});
+            mkBrowserTitleComponent({browserId: 'yabro', browserName: 'yabro'}, {tree});
```

Comment:
Does this line do anything? As far as I can tell we are just callling a function that returns a value and ignore the value.

Reply:
Yes. It calls a stub that is used in assert inside the component.